hbase_sql/src/main/java/com/splicemachine/ddl/ZooKeeperDDLWatchChecker.java

```diff
@@ -159,4 +180,37 @@ public void killDDLTransaction(String key){
        zkClient.deleteChangeNode(key);
     }
 
+
+    @Override
+    public void assignDDLDemarcationPoint(long txnId) throws Exception {
+        try{
+            lock.acquire();
+            String rootPath = SIDriver.driver().getConfiguration().getSpliceRootPath();
+            String ddlPath = rootPath+HConfiguration.DDL_PATH;
+            byte[] data = ZkUtils.getData(ddlPath);
+            long demarcationPoint = data != null ? Bytes.toLong(data) : 0;
+            if (txnId > demarcationPoint) {
+                data = Bytes.toBytes(txnId);
+                ZkUtils.setData(ddlPath, data, -1);
+                if (LOG.isDebugEnabled()) {
+                    SpliceLogUtils.debug(LOG, "Set DDL demarcation point to %d", txnId);
+                }
+            }
+        }
+        finally {
+            lock.release();
+        }
+    }
+
+    @Override
+    public long getDemarcationPoint() throws IOException{
+        String rootPath = SIDriver.driver().getConfiguration().getSpliceRootPath();
+        String ddlPath = rootPath+HConfiguration.DDL_PATH;
+        byte[] data = ZkUtils.getData(ddlPath);
+        long demarcationPoint = data != null ? Bytes.toLong(data) : 0;
+        if (LOG.isDebugEnabled()) {
+            SpliceLogUtils.debug(LOG, "Get DDL demarcation point %d from zookeeper", demarcationPoint);
```

Comment:
This only happens during startup right? Please increase the logging level to INFO